Dockerfile

```diff
@@ -17,7 +17,7 @@ ENV CHECKSTYLE_DIRECTORY  ${LINTERS_DIRECTORY}/checkstyle
 ENV DETEKT_VERSION  1.14.2
 ENV DETEKT_DIRECTORY  ${LINTERS_DIRECTORY}/detekt
 
-ENV PMD_VERSION  6.36.0
+ENV PMD_VERSION  6.37.0
```

Comment:
This is different from the pull request description.

> Updated PMD from 6.46.0 to 6.47.0.

What version of PMD should be?

Reply:
Sorry. I accidentally misprinted. 
> Updated PMD from 6.36.0 to 6.37.0.